mempool/cat/pool.go

```diff
@@ -171,16 +174,22 @@ func (txmp *TxPool) Has(txKey types.TxKey) bool {
 	return txmp.store.has(txKey)
 }
 
-// Get retrieves a transaction based on the key. It returns a bool
-// if the transaction exists or not
-func (txmp *TxPool) Get(txKey types.TxKey) (types.Tx, bool) {
+// GetTxByKey retrieves a transaction based on the key. It returns a bool
+// indicating whether transaction was found in the cache.
+func (txmp *TxPool) GetTxByKey(txKey types.TxKey) (types.Tx, bool) {
```

Comment:
This is a breaking change. It may be better to still have `Get` and make it wrap `GetTxByKey` if you want to change the name of the method